packages/ui/src/elements/ListDrawer/types.ts

```diff
@@ -35,10 +35,12 @@ export type UseListDrawer = (args: {
   React.FC<Pick<ListTogglerProps, 'children' | 'className' | 'disabled'>>, // toggler
   {
     closeDrawer: () => void
+    collectionSlugs: string[]
```

Comment:
This could be typed as `SanitizedCollectionSlug['slug']` or similar

Reply:
though `SanitizedCollectionConfig['slug']` is probably a better source of truth, we can change `SanitizedCollection.slug` with `CollectionSlug` later I think.